assets/css/_faq.scss

```diff
@@ -0,0 +1,53 @@
+.m-faq {
+  background: $color-bg-medium;
+
+  @media (min-width: 720px) {
+    height: 100%;
+    position: fixed;
+    width: 100%;
+  }
+}
+
+.m-faq__container {
+  background: $color-bg-light;
+  padding: 2rem;
+
+  @media (min-width: 720px) {
+    width: 50%;
+    height: 100%;
+    position: fixed;
+  }
+
+  @media (min-width: 1200px) {
+    width: 33%;
+  }
+}
+
+.m-faq__title {
+  @include font-header;
+  text-align: center;
+}
+
+.m-faq__section {
+  background-color: $white;
+  border-radius: 8px;
+  box-shadow: $subtle-box-shadow;
+  margin-bottom: 1rem;
+  padding: 0.5rem 1rem;
+}
+
+.m-faq__header {
+  @include font-emph;
+  font-weight: bold;
+}
+
+.m-faq__link {
+  background-color: $color-primary;
```

Comment:
This can include the button mixins from `_ui-kit.scss`.

Reply:
totally missed those, thanks